### PR TITLE
(Added)[PXN-3140] - Added paymentMethodRuleSet

### PR DESCRIPTION
**Pull request by jscaramal-meli**
## What have changed?
We've added paymentMethodRuleSet to init checkout request

## How to test:
You can add this line to paymentMethodSelector builder func on ExampleApp:

builder.setPaymentMehtodRuleSet(paymentMethodRuleSet: ["IGNORE_INSUFFICIENT_AM_BALANCE"])

And check that the field is being send.

### CODE REVIEW

**Review by jlainog-meli — COMMENTED** [review]
agrega el changelog antes de mergear

**Review by jlainog-meli — APPROVED** [review]

File: MercadoPagoSDK/MercadoPagoSDK/Core/PaymentMethodSelector/PXPaymentMethodSelector.swift [hunk 1]
```diff
@@ -55,6 +56,10 @@ public class PXPaymentMethodSelector: NSObject {
             self.paymentMethodBehaviours = paymentMethodBehaviours
         }
 
+        public func setPaymentMehtodRuleSet(paymentMethodRuleSet: [String]) {
```
Comment (mvarani) [inline MercadoPagoSDK/MercadoPagoSDK/Core/PaymentMethodSelector/PXPaymentMethodSelector.swift:59]: Tremendo typo en el nombre jaja. Le ponemos setPaymentMethodRuleSet?

**Review by jlainog-meli — DISMISSED** [review]

File: MercadoPagoSDK/MercadoPagoSDK/Core/PaymentMethodSelector/PXPaymentMethodSelector.swift (hunk 1 above)
Comment (jlainog-meli) [reply MercadoPagoSDK/MercadoPagoSDK/Core/PaymentMethodSelector/PXPaymentMethodSelector.swift]: Jajajaja, si si, arreglemos eso

Comment (jscaramal-meli) [reply MercadoPagoSDK/MercadoPagoSDK/Core/PaymentMethodSelector/PXPaymentMethodSelector.swift]: Solved, TY

**Review by mvarani — APPROVED** [review]

**Review by jlainog-meli — APPROVED** [review]

**Pull request merged**